public/app/features/dashboard-scene/panel-edit/VizPanelManager.tsx

```diff
@@ -76,7 +76,7 @@ export class VizPanelManager extends SceneObjectBase<VizPanelManagerState> {
   public static createFor(sourcePanel: VizPanel) {
     let repeatOptions: Pick<VizPanelManagerState, 'repeat' | 'repeatDirection' | 'maxPerRow'> = {};
 
-    const gridItem = sourcePanel.parent instanceof LibraryVizPanel ? sourcePanel.parent.parent : sourcePanel.parent;
+    const gridItem = sourcePanel?.parent instanceof LibraryVizPanel ? sourcePanel.parent.parent : sourcePanel?.parent;
```

Comment:
@axelavargas is this change intended? Seems like these test work without this change?

Reply:
good catch, not intended :)